web-bluetooth/reset-energy.html

```diff
@@ -0,0 +1,69 @@
+---
+feature_name: Reset Energy Expended Web Bluetooth
+chrome_version: 45
+feature_id: 5264933985976320
+---
+
+<p>The <a href="https://github.com/WebBluetoothCG/web-bluetooth">Web Bluetooth API</a> aims to discover and communicate with devices over the Bluetooth 4 wireless standard using the Generic Attribute Profile (GATT). It is currently only partially implemented in Chrome OS behind the experimental flag <code>chrome://flags/#enable-web-bluetooth</code>.</p>
+
+<p>This sample illustrates the use of the Web Bluetooth API to reset energy
+expended from a nearby Bluetooth Device advertising Heart Rate with Bluetooth
+Low Energy. You may want to try this demo with the BLE Peripheral Simulator
+App from the <a target="_blank"
+href="https://play.google.com/store/apps/details?id=io.github.webbluetoothcg.bletestperipheral">Google
+Play Store</a>.</p>
+
+<button>Reset Energy Expended</button>
+{% capture initial_output_content %}
+{% endcapture %}
+{% include output_helper.html initial_output_content=initial_output_content %}
+
+{% capture js %}
+function onButtonClick() {
+  'use strict';
+
+  log('Requesting Bluetooth Device...');
+  navigator.bluetooth.requestDevice({filters:[{services:['heart_rate']}]})
+  .then(device => {
+    log('> Found ' + device.name);
+    log('Connecting to GATT Server...');
+    return device.connectGATT();
+  })
+  .then(server => {
+    log('Getting Heart Rate Service...');
+    return server.getPrimaryService('heart_rate');
+  })
+  .then(service => {
+    if (!service) throw 'Heart Rate service not found';
+    log('Getting Heart Rate Control Point Characteristic...');
+    return service.getCharacteristic('heart_rate_control_point');
+  })
+  .then(characteristic => {
+    log('Writing Heart Rate Control Point Characteristic...');
+
+    // Writing 1 is the signal to reset energy expended.
+    let resetEnergyExpended = new Uint8Array([1]);
```

Comment:
@g-ortuno, we need to get notifications working so we can demonstrate that this had an effect. :)